### PR TITLE
Adds SQN model guide.

### PR DESCRIPTION
**Pull request by vbhv14**
- [x] Added SQN model guide.

### CODE REVIEW

**Comment by vbhv14** [comment]
@BP-Ent requesting your review/feedback.

**Review by jyaistMap — CHANGES_REQUESTED** [review]
Visually the guide looks good when build locally. 

In all links to the `References` section, please change _References_ to _references_.  The upper case _R_ renders the link inactive when built for the website.

File: guide/14-deep-learning/point_cloud_classification_using_sqn.ipynb [hunk 1]
```diff
@@ -0,0 +1,485 @@
+{
```
Comment (jyaistMap) [inline guide/14-deep-learning/point_cloud_classification_using_sqn.ipynb:1]: <p>Change the text for the actual link to the references from <em>#References</em> to _#references_ </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (jyaistMap) [inline guide/14-deep-learning/point_cloud_classification_using_sqn.ipynb:1]: <p>Change the hyperlink text from <em>#References</em> to _#references_ </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (jyaistMap) [inline guide/14-deep-learning/point_cloud_classification_using_sqn.ipynb:1]: <p>Change hyperlink text to the references from <em>#References</em> to _#references_ </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (jyaistMap) [inline guide/14-deep-learning/point_cloud_classification_using_sqn.ipynb:1]: <p>Change hyperlink text to the references from <em>#References</em> to _#references_ </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

**Review by vikashranjan26 — APPROVED** [review]
LGTM

**Comment by priyankatuteja** [comment]
> Visually the guide looks good when build locally.
> 
> * In all links to the `References` section, please change _References_ to _references_.  The upper case _R_ renders the link inactive when built for the website.

@jyaistMap  Why is this change suggested? Is that a requirement for all notebooks

**Review by priyankatuteja — APPROVED** [review]

**Pull request merged**